image_opts.go

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package containerd
+
+type imageOpts struct {
+	dcparameters map[string][][]byte
+}
+
+// ImageOpt allows callers to set options on the containerd
+type ImageOpt func(i *imageOpts) error
```

Comment:
Wondering if this should be specific to unpack..

Reply:
@crosbymichael ?

Reply:
specific to unpack

Reply:
Renamed to `imageUnpackOpts` in `image_opts.go`